packages/cli/src/Server.ts

```diff
@@ -1693,197 +1708,200 @@ class App {
 		// Updates existing credentials
 		this.app.patch(
 			`/${this.restEndpoint}/credentials/:id`,
-			ResponseHelper.send(
-				async (req: express.Request, res: express.Response): Promise<ICredentialsResponse> => {
-					const incomingData = req.body;
+			ResponseHelper.send(async (req: CredentialRequest.Update): Promise<ICredentialsResponse> => {
+				const { id: credentialId } = req.params;
 
-					const { id } = req.params;
+				const updateData = new CredentialsEntity();
+				Object.assign(updateData, req.body);
```

Comment:
just an idea, wdyt?
```suggestion
				const updateData = Object.assign(new CredentialsEntity(), req.body);
```

since we are doing this part quite often I am thinking if we should create a factory for it 🤔